src/components/Todo/Todo.tsx

```diff
@@ -49,8 +49,14 @@ function TodoApp(props: TodoAppProps) {
     setItems([...updatedItems, ...completedItems]);
   };
 
+  const [theme, setTheme] = useState("light-theme");
```

Comment:
As people will use `react-todo` as an external dependency. Can you expose a prop to control the theme? instead of using local state. Just make `light-theme` default value for the prop.